src/libraries/Microsoft.PowerFx.Core/Binding/ISetGlobalSymbols.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Collections.ObjectModel;
+using System.Text;
+using Microsoft.PowerFx.Core.IR.Symbols;
+
+namespace Microsoft.PowerFx.Core.Binding
+{
+    internal interface ISetGlobalSymbols
+    {
+        abstract void SetGlobalSymbols(IReadOnlyDictionary<string, IGlobalSymbol> globalSymbols = null);
```

Comment:
This might be a c# feature I'm unfamiliar with, what does it mean for an interface method to be `abstract`?

Reply:
I have been playing with default interface implementations and forgot removing this abstract

[https://docs.microsoft.com/en-gb/dotnet/csharp/whats-new/tutorials/default-interface-methods-versions](https://docs.microsoft.com/en-gb/dotnet/csharp/whats-new/tutorials/default-interface-methods-versions)